benchmark/Project.toml

```
@@ -1,12 +1,13 @@
 [deps]
 BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
 OrdinaryDiffEqLowStorageRK = "b0944070-b475-4768-8dec-fb6eb410534d"
-OrdinaryDiffEqTsit5 = "b1df2697-797e-41e3-8120-5422d3b24e4a"
 OrdinaryDiffEqRosenbrock = "43230ef6-c299-4910-a778-202eb28ce4ce"
+OrdinaryDiffEqTsit5 = "b1df2697-797e-41e3-8120-5422d3b24e4a"
 SummationByPartsOperators = "9f78cca6-572e-554e-b819-917d2f1cf240"
 
 [compat]
 BenchmarkTools = "1"
 OrdinaryDiffEqLowStorageRK = "1"
+OrdinaryDiffEqRosenbrock = "1"
 OrdinaryDiffEqTsit5 = "1"
 SummationByPartsOperators = "0.5.63"
```

```suggestion
OrdinaryDiffEqLowStorageRK = "1.1"
OrdinaryDiffEqRosenbrock = "1.3"
OrdinaryDiffEqTsit5 = "1.1"
SummationByPartsOperators = "0.5.79"
```
Let's use the same compat bounds as for the tests.